tests/dummy/app/pods/view/renderers/documentation/autocomplete.md

```diff
@@ -0,0 +1,259 @@
+## autocomplete
+
+This renderer provides an autocomplete input which allows one option to be selected.
```

Comment:
Expand on he differences between `select` and `autocomplete` please.

Reply:
Also, the rest of the docs is a dup to `select`. I'd hate to have to maintain 2 separate docs if the implementation for one is altered. Perhaps provide a link and mention that the API is the same as `select`.

Reply:
ya, i'll reference select. I didn't like duping it anyways